lib/subproviders/geth_api_double.js

```diff
@@ -390,6 +390,14 @@ GethApiDouble.prototype.personal_newAccount = function(password, callback) {
   callback(null, account.address);
 };
 
+GethApiDouble.prototype.personal_importRawKey = function(rawKey, password, callback) {
+  var account = this.state.createAccount({ secretKey: rawKey });
```

Comment:
This should probably be updated to make use of `options.defaultBalanceEther` (see latest README.md in `develop` branch)

Reply:
Actually, scratch that - we likely want this account to be added with a zero balance.